### PR TITLE
[fineoffsetweatherstation] Add channel for the sensors battery voltage

### PR DESCRIPTION
**Pull request by Andy2003**
for discussion see here: https://community.openhab.org/t/fine-offset-weather-station-binding-beta-and-discussion/134167/80

### CODE REVIEW

File: bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml [hunk 1]
```diff
@@ -4,6 +4,17 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
+	<channel-type id="battery-voltage">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>Voltage</label>
```
Comment (jlaur) [inline bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml:9]: **Battery** Voltage? Consider also adding a description.

File: bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml [hunk 2]
```diff
@@ -4,6 +4,17 @@
 	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
 	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
 
+	<channel-type id="battery-voltage">
```
Comment (jlaur) [inline bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml:7]: Consider if this should be marked as advanced:
```suggestion
	<channel-type id="battery-voltage" advanced="true">
```

**Review by jlaur — CHANGES_REQUESTED** [review]
Thank you! Two minor things to consider.

File: bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml (hunk 2 above)
Comment (Andy2003) [reply bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml:7]: I would not use advanced here, as it is a meaningful channel that should also be displayed directly to the user

Comment (jlaur) [reply bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml:7]: OK.

File: bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml [hunk 3]
```diff
@@ -6,7 +6,8 @@
 
 	<channel-type id="battery-voltage">
 		<item-type>Number:ElectricPotential</item-type>
-		<label>Voltage</label>
+		<label>Battery Voltage</label>
+		<description>The voltage of the battery</description>
```
Comment (jlaur) [inline bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/sensor.xml:10]: i18n properties file needs to be regenerated after this change.

**Comment by Hilbrand** [comment]
It is not allowed to create branches on the openHAB repo itself!

**Comment by Andy2003** [comment]
> It is not allowed to create branches on the openHAB repo itself!

Sorry, this was an oversight. should I delete the branch and create a new PR, or can we continue here?

**Review by jlaur — APPROVED** [review]
LGTM

**Comment by jlaur** [comment]
@Andy2003 - I have merged now and deleted the branch.

**Pull request merged**